### ISSUE TITLE
Repository name

### ISSUE THREAD

**Issue by mbukatov** [gluster/operator#27]
**Describe the bug**

Naming this repository just "operator" feels wrong, why is this isn't called "gluster-operator" instead?

**Steps to reproduce**
1. see the name of this repository

**Actual results**

The repository is named just "operator"

**Expected behavior**

The repository is named "gluster-operator"

**Additional context**

See examples of other operator projects named in a same way:

 * https://github.com/coreos/etcd-operator
 * https://github.com/coreos/prometheus-operator

**Comment by JohnStrunk** [comment]
Given that the repo is under the gluster org, it seems somewhat redundant to have gluster/gluster-operator.

Do you know of a situation where the existing name will present a problem?

**Comment by mbukatov** [comment]
> Do you know of a situation where the existing name will present a problem?

Someone forks it into own github profile and then looking just at the name isn't enough to understand what the repository is about. Similar problem happens when someone clones it into local machine (eg. if I were cloning this repo, I do `git clone git@github.com:gluster/operator.git gluster-operator` to help myself recalling what this directory is about later).

Imho naming the repository using generic term looks like a bad idea. We have `gluster-kubernetes` repo and not just `kubernetes` repo, or `gluster-ansible` and no just `ansible`.

**Comment by JohnStrunk** [comment]
@mbukatov After talking it over w/ the rest of the team, we've decided on a bit of a different path than "gluster-operator".

Following the "ant" theme of Gluster, one of the devs (@jarrpa) has been experimenting with an operator under the name "[anthill](https://github.com/jarrpa/anthill)", and we've decided to bring that name forward into the official project.

While this doesn't match your original request for gluster-operator, I hope it does at least satisfy your desire for a unique/memorable name.
